backend/django/create-user-model.mo.md

```diff
@@ -0,0 +1 @@
+Coroute lol
```

Comment:
*Croute (typo)

Reply:
You could put an emoji, be a bit more 2017!